main.go

```diff
@@ -197,6 +201,15 @@ func main() {
 
 	cacheRecorderReader := NewRestoreReader(cacheReader)
 
+	if bytesRead, err := ioutil.ReadAll(cacheRecorderReader); err == nil {
```

Comment:
If the [streamed uncompress fails](https://github.com/bitrise-steplib/steps-cache-pull/pull/19/files#diff-7ddfb3e035b42cd70649cc33393fe32cR251), we [restart the whole download](https://github.com/bitrise-steplib/steps-cache-pull/pull/19/files#diff-7ddfb3e035b42cd70649cc33393fe32cR256), that should be tracked as well.

Reply:
We decided not to log this